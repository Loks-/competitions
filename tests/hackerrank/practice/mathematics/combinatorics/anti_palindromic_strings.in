4
2 2
2 3
5 6
6 5
