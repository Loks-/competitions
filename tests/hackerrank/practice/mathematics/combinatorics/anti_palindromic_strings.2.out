1920
1620
